permissions/plugin-zap.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "zap"
+paths:
+- "org/jenkinsci/plugins/zap"
```

Comment:
Why did you strip the hyphen from this? Note that only the package name cannot handle the dash.